### ISSUE TITLE
Template selection missing

### ISSUE THREAD

**Issue by XueSheng-GIT** [nextcloud/android#8358]
### Steps to reproduce
1. Create new text document
2. Dialog for file creation appears

### Expected behaviour
- Available templates should be shown

### Actual behaviour
- Only filename can be entered, no template selection available (although templates are available and selectable if using the nextcloud webinterface!). See Screenshot.
![Nextcloud new document](https://user-images.githubusercontent.com/4077683/117255275-be17ab80-ae49-11eb-8b78-c12cc62fc9f4.jpg)

### Environment data
Android version: 10

Device model: Huawei Mate40Pro

Stock or customized system: Stock

Nextcloud app version: 3.16.0

Nextcloud server version:  21.0.1

Reverse proxy: Nginx

**Comment by fto77533** [comment]
yes تم الإرسال من هاتف Huawei الخاص بي-------- الرسالة الأصلية --------من:‏ XueSheng-GIT ***@***.***>التاريخ:‏ الخميس، ٦ مايو، ٢٠٢١ ١٠:٠٥إلى:‏ nextcloud/android ***@***.***>نسخة إلى:‏ Subscribed ***@***.***>الموضوع:‏ [nextcloud/android] Template selection missing (#8358)
Steps to reproduce
Create new text documentDialog for file creation appears
Expected behaviour
Available templates should be shown
Actual behaviour
Only filename can be entered, no template selection available (although templates are available and selectable if using the nextcloud webinterface!). See Screenshot.

Environment data
Android version: 10
Device model: Huawei Mate40Pro
Stock or customized system: Stock
Nextcloud app version: 3.16.0
Nextcloud server version:  21.0.1
Reverse proxy: Nginx

**Comment by AndyScherzinger** [comment]
@XueSheng-GIT @fto77533 which templates do you see on the server for _text-files_, since as you can see in the file name field it found a template für mark-down, hence the dummy file name + extension. The selection above the file name is only shown if the client retrieves more than one template from the server.

**Comment by XueSheng-GIT** [comment]
This "Empty file.md" doesn't seem to be related to available templates. Currently there's a "Rezept.md" and "Template.md" (see Screenshot).
![Templates_webif](https://user-images.githubusercontent.com/4077683/117263155-4601b380-ae52-11eb-9a7f-4b54c3a8dbc7.png)

I also removed the template folder on nextcloud and created a new one... even tried different names for the templates folder (just in case), but nothing seems to have an influence. Android client doesn't show me any templates.

**Comment by AndyScherzinger** [comment]
Thanks for the update and infos. @tobiasKaminsky and @juliushaertl any ideas why the templates don't show up on the client (guess it doesn't retrieve them?)

**Comment by juliushaertl** [comment]
Hm, let me check, because I thought we exposed them through the direct editing API that the client uses.

**Comment by XueSheng-GIT** [comment]
Just tested android client 3.16.1 RC1. Templates are still not found/available. Any workaround? I assume it must be working somehow!?

**Comment by XueSheng-GIT** [comment]
Issue is still present!

**Comment by szaimen** [comment]
I can reproduce that. Maybe it is a bug due to the usage of external storage?

**Comment by juliushaertl** [comment]
Still on my list to look into

**Comment by XueSheng-GIT** [comment]
Issue still present and not fixed yet.
